.github/workflows/e2e.yaml

```diff
@@ -42,33 +41,38 @@ jobs:
     steps:
     - name: Check out code into the Go module directory
       uses: actions/checkout@v4
-
     - name: Set up Go
       uses: actions/setup-go@v5
       with:
         go-version: "~1.21.3"
-
     - name: Cache tools
       uses: actions/cache@v4
       with:
         path: bin
         key: ${{ runner.os }}-${{ runner.arch }}-${{ hashFiles('Makefile') }}
-
     - name: Install tools
       run: make install-tools
-
-    - name: "run tests"
+    - name: Install chainsaw
+      uses: kyverno/action-install-chainsaw@07b6c986572f2abaf6647c85d37cbecfddc4a6ab # v0.1.3
+    - name: Prepare e2e tests
```

Comment:
I would prefer this to be part of `make install-tools`, and for that target to install chainsaw locally just like we do for every other tool necessary to run e2e tests.

Reply:
Can dependabot upgrade actions pinned to a specific commit?

Reply:
Yup, works perfectly fine.

Reply:
See https://github.com/kyverno/kyverno/pull/9715/files for example.

Reply:
Done.